README.md

```diff
@@ -606,6 +606,81 @@ Output is written to stdout.
       future obfuscation with --load_map.); default: "";
 ```
 
+## Indexing Verible C++ Code using Kythe
+
+The steps mentioned here can be generalized for indexing Bazel-based projects.
```

Comment:
Add a reference to the Kythe documentation where these instructions came from in case we ever need to update to follow upstream recommendations.